Packs/FeedOpenCTI/Integrations/FeedOpenCTI/README.md

```diff
@@ -0,0 +1,155 @@
+## Configure OpenCTI Feed on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for OpenCTI Feed.
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| apikey | API Key | True |
+| base_url | Base URL | True |
+| indicator_types | Indicators Type to fetch | True |
+| max_indicator_to_fetch | Max. indicators per fetch \(default is 500\) | False |
+| feed | Fetch indicators | False |
+| feedReputation | Indicator Reputation | False |
+| feedReliability | Source Reliability | True |
+| feedExpirationPolicy |  | False |
+| feedExpirationInterval |  | False |
+| feedFetchInterval | Feed Fetch Interval | False |
+| feedTags | Tags | False |
+| feedBypassExclusionList | Bypass exclusion list | False |
+| insecure | Trust any certificate \(not secure\) | False |
+| proxy | Use system proxy settings | False |
+
+#### Indicator type parameter
+Possible values that are supported in XSOAR and will be generated out of the box:
+
+| **Types** |
+| --- |
+| ALL |
+| User-Account |
+| Domain |
+| Email-Address| 
+| File-md5| 
+|File-sha1| |
+|File-sha256|
+|HostName| 
+|IPV4-Addr|
+|IPV6-Addr| 
+|Registry-Key-Value|
+|URL|
+ 
+The following types are supported in OpenCTI but are not supported out of the box in XSOAR. To pull these indicator types from OpenCTI you will need to either create a dedicated [classification and mapping](https://xsoar.pan.dev/docs/incidents/incident-classification-mapping) and/or create a corresponding indicator type in your XSOAR system.
+
+
+| **Types** |
+| --- |
+|autonomous-system|
+|cryptographic-key|
+|cryptocurrency-wallet|
+|email-subject|
+|directory|
+|file-name|
+|file-path|
+|mac-addr|
+|mutex|
+|pdb-path|
+|process|
+|registry-key-value|
+|user-agent|
+|windows-service-name|
+|windows-service-display-name|
+|windows-scheduled-task|
+|x509-certificate-issuer|
+|x509-certificate-serial-number|
+
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### opencti-get-indicators
+***
+Gets indicators from the feed.
+
+
+#### Base Command
+
+`opencti-get-indicators`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| limit | The maximum number of indicators to return per fetch. The default value is "50". | Optional | 
+| indicator_types | The indicator types to fetch. Possible values are: "ALL", "User-Account", "Domain", "Email-Address", "File-md5", "File-sha1", "File-sha256", "HostName", "IPV4-Addr", "IPV6-Addr", "Registry-Key-Value", "URL" That they are supported in XSOAR, or another unsupported type, a clear explanation and the list of all types appears in the README. The default is "ALL". | Optional | 
```

Comment:
This isn't really clear. What are you trying to convey after you list the possible values?